app.js

```diff
@@ -4,22 +4,28 @@ const {
   getReviews,
   getReviewByReviewId,
   getCommentsByReviewId,
+  postCommentOnReview,
 } = require("./controller");
 
 const app = express();
+app.use(express.json());
 
 app.get("/api/categories", getCategories);
 app.get("/api/reviews", getReviews);
 app.get("/api/reviews/:review_id", getReviewByReviewId);
 app.get("/api/reviews/:review_id/comments", getCommentsByReviewId);
 
+app.post("/api/reviews/:review_id/comments", postCommentOnReview);
+
 app.all("/*", (req, res) => {
   res.status(404).send({ msg: "Route not found" });
 });
 
 app.use((err, req, res, next) => {
   if (err.code === "22P02") {
     res.status(400).send({ msg: "Invalid ID" });
+  } else if (err.code === "23503") {
+    res.status(400).send({ msg: "Username unknown" });
```

Comment:
this psql error could be thrown by any faulty insert/update query, maybe an idea to make your error message less specific to account for this if adding more post requests in the future?